src/elements/content-sidebar/versions/VersionsItem.scss

```diff
@@ -36,28 +36,35 @@
 
 .bcs-VersionsItem-details {
     flex: 1 1 100%;
+    margin-right: 35px; // Avoid flowing under action toggle button
     overflow: hidden;
+
+    .bcs-VersionsItem-log {
```

Comment:
Please avoid nesting classes when possible, as it increases specificity and is usually not necessary with SUIT conventions.